src/common/regions/region_db.ts

```diff
@@ -48,7 +61,7 @@ class RegionDB {
   }
 
   all(): Region[] {
-    return this.regions;
+    return [...this.states, ...this.counties, ...this.metroAreas];
```

Comment:
I don't think it'll really be an issue, but maybe better to not include metro areas until they're really supported elsewhere